backend/database/user.js

```diff
@@ -45,6 +47,103 @@ const userInfo = async (req, res) => {
   }
 };
 
+// NOTE: re-using admin's /invite implementation and database
+const forgotPassword = async (req, res) => {
+  const email = req.body.email;
+
+  // Check if email is valid
+  const data = await db.query(`SELECT * FROM users WHERE email = $1;`, [email]);
+  const arr = data.rows;
+  if (arr.length === 0) {
+    return res.status(400).json({ message: `Email doesn't exist. Please register first.` });
+  }
+
+  const zid = await db.query(`SELECT zid FROM users WHERE email = $1;`, [email]);
+
+  // NOTE: re-using database because of tight deadlines
+  // token stores max size 32. Chose to use zid over email to fit length.
+  const token = btoa(Math.floor(Math.random() * 99999999999999 + 1) + zid.rows[0].zid);
```

Comment:
In generating the token, I believe we should stick maybe to the same unique id (uuidv4)?